src/fft/polynomial.rs

```diff
@@ -113,30 +118,35 @@ impl Polynomial {
         }
     }
 
-    /// Evaluates a [`Polynomial`] at a given point in the field.
-    pub(crate) fn evaluate(&self, point: &BlsScalar) -> BlsScalar {
+    /// Evaluates a [`Polynomial`] at a given scalar in the field.
```

Comment:
even when is the common way to say it, indeed was a bit confusing to talk about 'points' in this context, as it could seem that it refers to a point in the curve. However, I would not put 'scalar' here as it is not a scalar (even when the type is). I would say 'value'.